internal/code-generator/assets/colors.go.tmpl

```diff
@@ -0,0 +1,7 @@
+package data
+
+var LanguagesColor = map[string]string{
```

Comment:
@lafriks here and below, could you also please make sure all exported functions/variables are documented? Thanks!